precise/scripts/train.py

```diff
@@ -85,9 +85,11 @@ def __init__(self, args):
         self.model = create_model(args.model, params)
         self.train, self.test = self.load_data(self.args)
 
-        from keras.callbacks import ModelCheckpoint, TensorBoard
+        from keras.callbacks import ModelCheckpoint, TensorBoard, EarlyStopping, ReduceLROnPlateau
```

Comment:
[TensorBoard unused]

Reply:
Addressed this is latest commit 'General cleanup for merge'.